lib/solveEquation/stepThrough.js

```diff
@@ -260,6 +260,7 @@ function logSteps(equationStatus) {
   if (equationStatus.substeps.length > 0) {
     // eslint-disable-next-line
     console.log('\n substeps: ');
+    // eslint-disable-next-line
     equationStatus.substeps.forEach(substep => logSteps);
```

Comment:
oh actually this should just be ` equationStatus.substeps.forEach(logSteps);` which fixes the lint problem

(my bad)